include/librealsense2/h/rs_device.h

```diff
@@ -80,6 +80,23 @@ int rs2_supports_device_info(const rs2_device* device, rs2_camera_info info, rs2
  */
 void rs2_hardware_reset(const rs2_device * device, rs2_error ** error);
 
+/**
+* Build raw data command from opcode, parameters and data.
+* The result can be used as raw_data_to_send parameter in send_and_receive_raw_data
+* \param[in]  device                    RealSense device to send data to
+* \param[in]  opcode                    Commad opcode
+* \param[in]  param1                    First input parameter
+* \param[in]  param2                    Second parameter
+* \param[in]  param3                    Third parameter
+* \param[in]  param4                    Fourth parameter
+* \param[in]  data                      Input Data (up to 1024 bytes)
+* \param[in]  size_of_data              Size of input data in bytes
+* \param[out] error                     If non-null, receives any error that occurs during this call, otherwise, errors are ignored
+* \return                               rs2_raw_data_buffer which includes raw command
+*/
+const rs2_raw_data_buffer* rs2_build_raw_data(rs2_device* device, unsigned opcode, unsigned param1, unsigned param2,
```

Comment:
I just realized "raw data" is used in many contexts in our APIs. Therefore the name is not specific enough.
How about `rs2_build_raw_data_command`?

Reply:
Or `rs2_build_debug_protocol_command`?

Reply:
I think the first one (**rs2_build_raw_data_command**) is more suitable, since we will use the result as parameter for "rs2_send_recieve_raw_data"
so, the "raw_data" is an important term.